presto-hive/src/main/java/com/facebook/presto/hive/HiveUtil.java

```diff
@@ -332,7 +332,7 @@ private static Deserializer createDeserializer(Class<? extends Deserializer> cla
     private static void initializeDeserializer(Deserializer deserializer, Properties schema)
     {
         try {
-            deserializer.initialize(null, schema);
+            deserializer.initialize(new Configuration(), schema);
```

Comment:
It seemed a bit questionable to me to pass in a new default configuration, but 
1. not all of the callers had access to an existing configuration to thread through
2. The thrift deserializer doesn't really use the configuration variables.  It just uses the configuration to load a class, so it doesn't really matter what it is.
3. Passing in a default configuration results in the same behavior as passing in null to the LazySimpleSerde which is what hive tables were using by default.